src/clj_kondo/core.clj

```diff
@@ -18,7 +18,12 @@
   subject to change."
   [{:keys [:config :findings :summary :analysis]}]
   (let [output-cfg (:output config)
-        fmt (or (:format output-cfg) :text)]
+        fmt (or (:format output-cfg) :text)
```

Comment:
It's unclear to me why this code was moved.

Reply:
Sorry, I'll put this back... it was part of an intermediate refactoring where the logic for when to show the analysis got complicated and I didn't want to duplicate it. Now, the presence or absence of `analysis` is all that's needed so it'll be ok to duplicate.